profiles/package.mask

```diff
@@ -33,6 +33,13 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2021-12-20)
+# Requires GCC 8, security vulnerabilities, switch to CUDA 11,
+# consider using clang+NVPTX backend for sm_30 cards.
+# Removal in 90 days, bug #721808, #760708.
+<dev-util/nvidia-cuda-sdk-11
+<dev-util/nvidia-cuda-toolkit-11
```

Comment:
Please don't drop this yet, at least not until there's a fix for https://bugs.gentoo.org/823882. I needed to downgrade CUDA to 10.2 on a machine recently because of that.

Reply:
maybe help fix it?

Reply:
I did try to fix it, and didn't get anywhere. This problem with cuda-gdb is something that likely needs to be fixed by nvidia.

Reply:
have you reported it like all of the several bugs I have reported in nvcc?

Reply:
Didn't have time last year. I will try some debugging again now that I'm back from vacations, and file a bug if I can't fix it. Still, just keeping 10.2 as is for a little while longer at least gives people that need to debug CUDA code a way to do so.

Reply:
Filed a bug and in 11.6.1, which I just pushed, cuda-gdb works again.

Reply:
so we can finally sunset CUDA 10?

Reply:
There's still a small issue with CUDA and some headers of GCC, but there's an easy workaround for that. I think we can drop CUDA 10 after a couple of weeks this new version is in the tree.

Reply:
which GCC header issues? The `<functional>` stuff? That has been fixed in 11.6.1 (after I reported it)

Reply:
No, `serializeintrin.h`.